RecoTracker/TkSeedingLayers/interface/SeedingLayerSetsBuilder.h

```diff
@@ -32,9 +34,15 @@ class SeedingLayerSetsBuilder {
   static void fillDescriptions(edm::ParameterSetDescription& desc);
 
   unsigned short numberOfLayers() const { return theLayers.size(); }
+  unsigned short numberOfLayerSets() const { return theNumberOfLayersInSet > 0 ? theLayerSetIndices.size()/theNumberOfLayersInSet : 0; }
+  std::vector<SeedingLayerId> layers() const; // please call at most once per job per client
+  SeedingLayerSetsLooper seedingLayerSetsLooper() const { return SeedingLayerSetsLooper(theNumberOfLayersInSet, &theLayerSetIndices); }
+
+  const std::vector<SeedingLayerSetsHits::LayerSetIndex>& layerSetIndices() const { return theLayerSetIndices; }
+
+  std::vector<std::vector<SeedingLayerId> > layerSets() const; // please call at most once per job per client
```

Comment:
where is this implemented?

Reply:
Likely leftover from the prototyping phase (in the end `layers()` was more useful), will remove.